react-native/ios/ReactNativeLoginKitDemo-tvOS/Info.plist

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
TODO for me, to remove tvOS scaffold since we do not want to confuse that we support it

Reply:
Can you send a commit for this then?